src/config/constants.ts

```diff
@@ -1,5 +1,8 @@
-export const DEFAULT_EOL_REPORT_URL = 'https://eol-report-card.stage.apps.herodevs.io';
+export const EOL_REPORT_URL = '';
+export const GRAPHQL_HOST = 'https://api.nes.herodevs.com';
 
-export function getEolReportUrl() {
-  return process.env.EOL_REPORT_URL || DEFAULT_EOL_REPORT_URL;
-}
+export const config = {
+  eolReportUrl: process.env.EOL_REPORT_URL || EOL_REPORT_URL,
+  graphqlHost: process.env.GRAPHQL_HOST || GRAPHQL_HOST,
+  graphqlPath: '/graphql',
```

Comment:
```suggestion
  graphqlPath: '/graphql',
```
This doesnt seem something that regards config

Reply:
It should be configurable, I'll update it accordingly so it can be overridden by process.env though

Reply:
Done [f330b7d](https://github.com/herodevs/cli/pull/224/commits/f330b7d779753bc22de33a766c89c817fe8fdb90)